syntax = "proto3";

enum LicenseStatus {
  APPROVE = 0;
  WAITING = 1;
  DUPLICATE = 2;
}


message LicenseRegistrationRequest {
  string reg_user_name = 1;
  string reg_user_tel = 2;
  string server_type = 3;
  string hardware_uuid = 4;
  string machine_id = 5;
}

message LicenseRequest {
  string hardware_uuid = 4;
  string machine_id = 5;
}

message LicenseResponse {
  LicenseStatus status = 1;
  string license_data = 2; /* 미리 승인되어 있으면 데이터 채우고 아니면 제외*/
}